### DISCUSSION TITLE
Can I set directivity of sound source?

### DISCUSSION THREAD

**Discussion by TomohisaKato** [Ifsttar/I-Simpa#314] [Q&A]
I want to set directivity of sound source. How can do it?

**Comment by ldaniel-jmz** [comment]
Right now the only way is by importing the directivity files itself. I-Simpa uses tabular text format directivity as specified by [Ease](http://ease.afmg.eu/index.php/documents.html) (export format).

But exporting this files is complicated, I'm also waiting for an alternative to this.
